frameworks/helloworld/tests/test_zzzrecovery.py

```diff
@@ -296,3 +269,59 @@ def test_config_update_then_zk_killed():
     sdk_cmd.kill_task_with_pattern('zookeeper')
     sdk_tasks.check_tasks_updated(config.SERVICE_NAME, 'hello', hello_ids)
     config.check_running()
+
+
+@pytest.mark.sanity
+def test_pod_replace():
+    world_ids = sdk_tasks.get_task_ids(config.SERVICE_NAME, 'world-0')
+
+    # get current agent id (TODO: uncomment if/when agent is guaranteed to change in a replace operation):
+    # jsonobj = sdk_cmd.svc_cli(config.PACKAGE_NAME, config.SERVICE_NAME, 'pod info world-0', json=True, print_output=False)
+    # old_agent = jsonobj[0]['info']['slaveId']['value']
+
+    jsonobj = sdk_cmd.svc_cli(config.PACKAGE_NAME, config.SERVICE_NAME, 'pod replace world-0', json=True)
+    assert len(jsonobj) == 2
+    assert jsonobj['pod'] == 'world-0'
+    assert len(jsonobj['tasks']) == 1
+    assert jsonobj['tasks'][0] == 'world-0-server'
+
+    sdk_tasks.check_tasks_updated(config.SERVICE_NAME, 'world-0', world_ids)
+    config.check_running()
+
+    # check agent moved (TODO: uncomment if/when agent is guaranteed to change (may randomly move back to old agent))
```

Comment:
I don't think replace will ever require movement to a new agent.  I'd just drop this code.